app/src/main/java/com/jjswigut/matters/ui/EditMatterFragment.kt

```diff
@@ -77,23 +79,17 @@ class EditMatterFragment : BaseFragment() {
         }
     }
 
-    // Fix the input part
-    private fun input(): Matter {
-        var title = binding.titleInput.text.toString().trim()
-        var content = binding.contentInput.text.toString().trim()
-        matter = Matter(title, content)
```

Comment:
this was part of the problem with editing an existing matter - every time you called this method, you were setting matter to be a new instance which deleted the old ID meaning we no longer have reference to it.

I didn't think this function served a ton of use after the fact.